### PR TITLE
solution 1.0

### PR DESCRIPTION
**Pull request by aliliolek**

### CODE REVIEW

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Provide a demo link, please

**Comment by aliliolek** [comment]
https://aliliolek.github.io/js_task_fix_form_DOM/

File: src/index.html [hunk 1]
```diff
@@ -11,42 +11,44 @@
       <form>
         <div class="field">
           <input type="text" name="firstName"
-                 class="field-text" autocomplete="off"
-                 id="sign-up-name" autofocus>
+            class="field-text" autocomplete="off"
+            id="sign-up-name" autofocus>
```
Comment (polosanya) [inline src/index.html:15]:
```suggestion
            class="field-text" 
            autocomplete="off"
            id="sign-up-name" 
            autofocus
          >
```

File: src/index.html [hunk 2]
```diff
@@ -11,42 +11,44 @@
...
         </div>
         <div class="field">
           <input type="text" name="lastName"
-                 class="field-text" id="sign-up-last-name"
-                 autocomplete="off">
+           class="field-text" id="sign-up-last-name"
+           autocomplete="off">
```
Comment (polosanya) [inline src/index.html:20]: Please fix all cases

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+const inputs = document.querySelectorAll('input');
+
+inputs.forEach(el => {
+  el.placeholder = el.name.replace(/[A-Z]/g, letter => ` ${letter}`);
+  el.placeholder = el.placeholder[0].toUpperCase() + el.placeholder.slice(1);
+
+  el.insertAdjacentHTML('beforebegin', `
+<label for="${el.id}" class="field-label">
+  ${el.name.replace(/[A-Z]/g, letter => ` ${letter}`).toUpperCase()}:
+</label>`);
```
Comment (polosanya) [inline src/scripts/main.js:13]:
```suggestion
  el.insertAdjacentHTML('beforebegin', `
    <label for="${el.id}" class="field-label">
      ${el.name.replace(/[A-Z]/g, letter => ` ${letter}`).toUpperCase()}:
    </label>
  `);
```
Also. would be great to create a separate variable for formatted name

**Review by polosanya — CHANGES_REQUESTED** [review]
Let's improve your code a bit!
Also, there is no need to add all the files to the commit, you can add only content of src folder

**Comment by aliliolek** [comment]
It is no longer valid

**Pull request closed without merging**